### DISCUSSION TITLE
Deauthentication/Disasoiciation Frames

### DISCUSSION THREAD

**Discussion by onurbi** [nzymedefense/nzyme#604] [Ideas]
Hi,
the new analysis of Deauthentication/Disasoiciation Frames show me when and how often such frames appeared in my network.

Is it planned to make it possible to show to what clients this frames have been directed?

This would it make possible to correlate a strange behaviour observed with a certain client.

Regards,

Onurbi

**Comment by lennartkoopmann** [comment]
Yes, this is a potential future feature. For now, forwarding to a system like Graylog will give you this ability.

**Reply by onurbi** [reply]
That is indeed a reason to try Graylog!
Is a Raspi with 2GB RAM able to run it? Sometimes nzyme let it swap a little already.

**Reply by lennartkoopmann** [reply]
You'll likely need something bigger. The SD card will become an issue, too. A 8GB RPI 4 should be able to run it.
